rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    person.rentals << self
+    @book = book
+    book.rentals << self
+  end
+end
```

Comment:
Hi, Nice Job so far. **Please make sure that the person has many rentals**: 
- What you need to do is to create the ` has-many` side of `Person` and `Rental`:  Rentals should be initialized([LINK TO EXAMPLE](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)). A method for adding rentals([LINK TO EXAMPLE](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship))
- **HINT**: This must be done inside `Person` class